beacon-chain/rpc/eth/light-client/handlers_test.go

```diff
@@ -534,210 +495,195 @@ func TestLightClientHandler_GetLightClientUpdatesByRangeDeneb(t *testing.T) {
 	var resp structs.LightClientUpdatesByRangeResponse
 	err = json.Unmarshal(writer.Body.Bytes(), &resp.Updates)
 	require.NoError(t, err)
-	var respHeader structs.LightClientHeaderDeneb
-	err = json.Unmarshal(resp.Updates[0].Data.AttestedHeader, &respHeader)
-	require.NoError(t, err)
 	require.Equal(t, 1, len(resp.Updates))
+	require.NoError(t, err)
```

Comment:
same here and other places I didn't understood why are we checking err here again ?